src/flash/com/longtailvideo/jwplayer/player/Player.as

```diff
@@ -63,16 +64,36 @@ public class Player extends Sprite implements IPlayer {
         // e.state can be ThrottleType.THROTTLE, ThrottleType.PAUSE, or ThrottleType.RESUME
         // in Chrome we only get 'throttle' and 'resume' for offscreen and power-save throttling
         var state:String = e['state'] as String;
-        if (state === 'resume') {
-            // After resume, we're past Chrome Power Save
-            // stop listening to the off-screen throttle events
-            this.removeEventListener('throttle', flashThrottled);
+
+        if (state !== 'resume') {
+            _model.removeEventListener(MediaEvent.JWPLAYER_MEDIA_TIME, removeThrottleListener);
+            // Ignore throttle events for players ignored by Chrome's heuristics
+            var audioPlayer:Boolean = RootReference.stage.stageWidth < 6 && RootReference.stage.stageHeight < 6;
+            var contentPlayer:Boolean = RootReference.stage.stageWidth > 400 && RootReference.stage.stageHeight > 300;
+            if (audioPlayer || contentPlayer) {
+                return;
+            }
+            // remove listener after time event
+            _model.addEventListener(MediaEvent.JWPLAYER_MEDIA_TIME, removeThrottleListener);
         }
+
         SwfEventRouter.triggerJsEvent('throttle', {
             state: state
         });
     }
 
+    private function removeThrottleListener(e:MediaEvent):void {
+        // After a time event has been received, we're past Chrome Power Save

```

Comment:
Power saving will occur on the first rendered frame if at all. This time event is the closest thing we have to a consistent first frame.